<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QMainWindow" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>472</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MP3 Tag Editor</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabsMode">
      <property name="minimumSize">
       <size>
        <width>821</width>
        <height>400</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>0</width>
        <height>400</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabFile">
       <attribute name="title">
        <string>Single File</string>
       </attribute>
       <widget class="QGroupBox" name="boxTag">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>1</y>
          <width>461</width>
          <height>357</height>
         </rect>
        </property>
        <property name="title">
         <string>Tag Info</string>
        </property>
        <widget class="QLabel" name="labelTrack">
         <property name="geometry">
          <rect>
           <x>48</x>
           <y>52</y>
           <width>53</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Track #</string>
         </property>
        </widget>
        <widget class="TextEdit" name="editTrack">
         <property name="geometry">
          <rect>
           <x>104</x>
           <y>52</y>
           <width>53</width>
           <height>25</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="lineWrapMode">
          <enum>QTextEdit::NoWrap</enum>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="labelDisc">
         <property name="geometry">
          <rect>
           <x>176</x>
           <y>52</y>
           <width>45</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Disc #</string>
         </property>
        </widget>
        <widget class="TextEdit" name="editDisc">
         <property name="geometry">
          <rect>
           <x>224</x>
           <y>52</y>
           <width>53</width>
           <height>25</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="lineWrapMode">
          <enum>QTextEdit::NoWrap</enum>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="labelBPM">
         <property name="geometry">
          <rect>
           <x>356</x>
           <y>52</y>
           <width>41</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>BPM</string>
         </property>
        </widget>
        <widget class="TextEdit" name="editBPM">
         <property name="geometry">
          <rect>
           <x>400</x>
           <y>52</y>
           <width>53</width>
           <height>25</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="lineWrapMode">
          <enum>QTextEdit::NoWrap</enum>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="labelTitle">
         <property name="geometry">
          <rect>
           <x>64</x>
           <y>80</y>
           <width>37</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Title</string>
         </property>
        </widget>
        <widget class="TextEdit" name="editTitle">
         <property name="geometry">
          <rect>
           <x>104</x>
           <y>80</y>
           <width>349</width>
           <height>25</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="lineWrapMode">
          <enum>QTextEdit::NoWrap</enum>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
        <widget class="TextEdit" name="editArtist">
         <property name="geometry">
          <rect>
           <x>104</x>
           <y>108</y>
           <width>349</width>
           <height>25</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="lineWrapMode">
          <enum>QTextEdit::NoWrap</enum>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
        <widget class="TextEdit" name="editAlbum">
         <property name="geometry">
          <rect>
           <x>104</x>
           <y>136</y>
           <width>349</width>
           <height>25</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="lineWrapMode">
          <enum>QTextEdit::NoWrap</enum>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
        <widget class="TextEdit" name="editAArtist">
         <property name="geometry">
          <rect>
           <x>104</x>
           <y>164</y>
           <width>349</width>
           <height>25</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="lineWrapMode">
          <enum>QTextEdit::NoWrap</enum>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="labelArtist">
         <property name="geometry">
          <rect>
           <x>56</x>
           <y>108</y>
           <width>45</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Artist</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelAlbum">
         <property name="geometry">
          <rect>
           <x>52</x>
           <y>136</y>
           <width>49</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Album</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelAArtist">
         <property name="geometry">
          <rect>
           <x>16</x>
           <y>164</y>
           <width>89</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Album Artist</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboTag">
         <property name="geometry">
          <rect>
           <x>102</x>
           <y>24</y>
           <width>101</width>
           <height>26</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="labelTag">
         <property name="geometry">
          <rect>
           <x>72</x>
           <y>24</y>
           <width>29</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Tag</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelYear">
         <property name="geometry">
          <rect>
           <x>68</x>
           <y>192</y>
           <width>33</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Year</string>
         </property>
        </widget>
        <widget class="TextEdit" name="editYear">
         <property name="geometry">
          <rect>
           <x>104</x>
           <y>192</y>
           <width>53</width>
           <height>25</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="lineWrapMode">
          <enum>QTextEdit::NoWrap</enum>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="labelGenre">
         <property name="geometry">
          <rect>
           <x>228</x>
           <y>192</y>
           <width>45</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Genre</string>
         </property>
        </widget>
        <widget class="GenreBox" name="comboGenre">
         <property name="geometry">
          <rect>
           <x>276</x>
           <y>192</y>
           <width>141</width>
           <height>25</height>
          </rect>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
        <widget class="TextEdit" name="editComment">
         <property name="geometry">
          <rect>
           <x>104</x>
           <y>220</y>
           <width>349</width>
           <height>65</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhMultiLine</set>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="lineWrapMode">
          <enum>QTextEdit::WidgetWidth</enum>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="labelComment">
         <property name="geometry">
          <rect>
           <x>32</x>
           <y>220</y>
           <width>69</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Comment</string>
         </property>
        </widget>
        <widget class="TextEdit" name="editPublisher">
         <property name="geometry">
          <rect>
           <x>104</x>
           <y>288</y>
           <width>349</width>
           <height>25</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="lineWrapMode">
          <enum>QTextEdit::NoWrap</enum>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="labelPublisher">
         <property name="geometry">
          <rect>
           <x>32</x>
           <y>288</y>
           <width>69</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Publisher</string>
         </property>
        </widget>
        <widget class="TextEdit" name="editComposer">
         <property name="geometry">
          <rect>
           <x>104</x>
           <y>316</y>
           <width>349</width>
           <height>25</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="lineWrapMode">
          <enum>QTextEdit::NoWrap</enum>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="labelComposer">
         <property name="geometry">
          <rect>
           <x>28</x>
           <y>316</y>
           <width>73</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Composer</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelBoxGenre">
         <property name="geometry">
          <rect>
           <x>420</x>
           <y>196</y>
           <width>33</width>
           <height>16</height>
          </rect>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="labelTagOffset">
         <property name="geometry">
          <rect>
           <x>208</x>
           <y>24</y>
           <width>89</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>@ %u bytes</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="boxInfo">
        <property name="geometry">
         <rect>
          <x>476</x>
          <y>224</y>
          <width>333</width>
          <height>133</height>
         </rect>
        </property>
        <property name="title">
         <string>MPEG Info</string>
        </property>
        <widget class="QLabel" name="labelNameOffset">
         <property name="geometry">
          <rect>
           <x>8</x>
           <y>48</y>
           <width>89</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>First Frame @</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelNameFrames">
         <property name="geometry">
          <rect>
           <x>8</x>
           <y>72</y>
           <width>89</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Frames Total:</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelNameLen">
         <property name="geometry">
          <rect>
           <x>8</x>
           <y>96</y>
           <width>89</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Track Lenght:</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelBitrate">
         <property name="geometry">
          <rect>
           <x>8</x>
           <y>20</y>
           <width>61</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>320 kbps</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelMode">
         <property name="geometry">
          <rect>
           <x>164</x>
           <y>20</y>
           <width>81</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Joint Stereo</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelEmphasis">
         <property name="geometry">
          <rect>
           <x>264</x>
           <y>20</y>
           <width>65</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>CCIT J.17</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelOffset">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>48</y>
           <width>165</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>41235675 bytes (+Xing)</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelFrames">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>72</y>
           <width>45</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>17000</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelLen">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>96</y>
           <width>133</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>1 hour 15 min 37 sec</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelSamplingRate">
         <property name="geometry">
          <rect>
           <x>88</x>
           <y>20</y>
           <width>65</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>44100 Hz</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="boxTagEx">
        <property name="geometry">
         <rect>
          <x>476</x>
          <y>0</y>
          <width>333</width>
          <height>221</height>
         </rect>
        </property>
        <property name="title">
         <string>More Tag Info</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tabBatch">
       <attribute name="title">
        <string>Batch Mode</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>845</width>
     <height>22</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy_Tag"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionCopy_Tag"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionCopy_Tag">
   <property name="text">
    <string>Copy Tag</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TextEdit</class>
   <extends>QTextEdit</extends>
   <header>textedit.h</header>
  </customwidget>
  <customwidget>
   <class>GenreBox</class>
   <extends>QComboBox</extends>
   <header>textedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
